### PR TITLE
website: Add manual deployment section

### PR DESCRIPTION
**Pull request by kfirtoledo**
Add manual deployment without using the CLI.

### CODE REVIEW

**Review by michalmalka — APPROVED** [review]
LGTM

File: website/content/en/docs/tasks/operator.md [hunk 1]
```diff
@@ -107,3 +106,48 @@ The `deploy peer` command has the following flags:
 - **tag:** This field determines the version of project images to pull. By default, it uses the `latest` version.
 
 All the flags are mapped to corresponding field in the ClusterLink custom resource.
+
+## Manual Deployment without CLI
+
+To deploy the ClusterLink project without using the CLI, follow the instructions below:
```
Comment (elevran) [inline website/content/en/docs/tasks/operator.md:112]: nit: Please avoid using `project` etc. It should be enough to say `To deploy ClusterLink without...`

File: website/content/en/docs/tasks/operator.md [hunk 2]
```diff
@@ -107,3 +106,48 @@ The `deploy peer` command has the following flags:
...
+
+1. Download the configuration files (CRDs, operator RBACs, and deployment) from GitHub:
+
+    ```sh
+    git clone git@github.com:clusterlink-net/clusterlink.git
+    ```
+
+1. Deploy ClusterLink CRDs:
```
Comment (elevran) [inline website/content/en/docs/tasks/operator.md:120]: nit: I would use `install` not deploy, but maybe that's just me

File: website/content/en/docs/tasks/operator.md [hunk 3]
```diff
@@ -107,3 +106,48 @@ The `deploy peer` command has the following flags:
...
+
+    ```sh
+    kubectl apply --recursive -f  ./clusterlink/config/crds
+    ```
+
+1. Deploy the ClusterLink operator:
+
+    ```sh
+    kubectl apply --recursive -f  ./clusterlink/config/operator
```
Comment (elevran) [inline website/content/en/docs/tasks/operator.md:129]: unless these are all autogenerated, consider grouping in a single file and not using recursive

File: website/content/en/docs/tasks/operator.md [hunk 4]
```diff
@@ -107,3 +106,48 @@ The `deploy peer` command has the following flags:
...
+    ```
+
+1. Convert the peer and fabric certificates to secrets:
+
+    ```sh
+    export CRT=<path to fabric certificates folder>
```
Comment (elevran) [inline website/content/en/docs/tasks/operator.md:135]: nit: `CERTS` may be a better name

File: website/content/en/docs/tasks/operator.md [hunk 5]
```diff
@@ -107,3 +106,48 @@ The `deploy peer` command has the following flags:
...
+    kubectl create secret generic cl-fabric -n clusterlink-system --from-file=ca=$CRT/cert.pem
```
Comment (elevran) [inline website/content/en/docs/tasks/operator.md:136]: please use TLS secrets if possible.
If it also requires code changes, we should do the whole change in a different PR

File: website/content/en/docs/tasks/operator.md [hunk 6]
```diff
@@ -107,3 +106,48 @@ The `deploy peer` command has the following flags:
 - **tag:** This field determines the version of project images to pull. By default, it uses the `latest` version.
 
 All the flags are mapped to corresponding field in the ClusterLink custom resource.
+
+## Manual Deployment without CLI
+
+To deploy the ClusterLink project without using the CLI, follow the instructions below:
+
+1. Download the configuration files (CRDs, operator RBACs, and deployment) from GitHub:
+
```
Comment (elevran) [inline website/content/en/docs/tasks/operator.md:115]: Can one get these from the release installation files instead of cloning the entire repo from git? Or at least from specific BLOB paths on GitHub?

Instructions should be directed to a user, not a developer.

**Review by elevran — APPROVED** [review]

File: website/content/en/docs/tasks/operator.md (hunk 3 above)
Comment (kfirtoledo) [reply website/content/en/docs/tasks/operator.md:129]: part of them are autogenerated.

File: website/content/en/docs/tasks/operator.md (hunk 5 above)
Comment (kfirtoledo) [reply website/content/en/docs/tasks/operator.md:136]: Should be part of https://github.com/clusterlink-net/clusterlink/issues/509

File: website/content/en/docs/tasks/operator.md (hunk 6 above)
Comment (kfirtoledo) [reply website/content/en/docs/tasks/operator.md:115]: I didn't find an easy way to do that (just cloning the config folder https://github.com/clusterlink-net/clusterlink/tree/main/config)
